src/services/tx/txSender.ts

```diff
@@ -176,6 +178,8 @@ export const dispatchTxSigning = async (
     txDispatch(TxEvent.SIGN_FAILED, { txId, tx: safeTx, error: error as Error })
     throw error
   }
+
+  trackEvent(WALLET_EVENTS.OFF_CHAIN_SIGNATURE)
```

Comment:
I asked you to subscribe to a tx event to not couple tracking with the sending logic, and you just moved it to another file but to the same effect.

Reply:
To explain why events are better for this: say, you're testing txSender. Now that tracking is added here, the tests should also cover that, mixing two completely separate subsystems: sender and analytics. Whereas with events it would be a completely separate module with its own tests.

Reply:
I'll adjust it. I didn't follow your original comment correct.